src/style.css

```diff
@@ -1,3 +1,71 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0 50px;
+}
+
+.logo__img {
+  display: block;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  color: #000;
+  text-decoration: none;
+  display: block;
+  line-height: 60px;
+  font-weight: 500;
+  font-size: 14px;
+  position: relative;
+}
+
+/* .nav__link {
+  display: block;
+  padding: 23px 0;
+  line-height: 14.06px;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #000;
+  margin: 0 auto;
+} */
+
+.nav__link:hover {
+  color: #00acdc;
```

Comment:
The color `#00acdc` should be defined as a CSS variable according to the task requirements. Consider adding a CSS variable, for example, `--blue-color: #00acdc;` and use it here as `color: var(--blue-color);`.